docker/cygnus-ngsi-ld/docker-compose.yml

```
@@ -40,7 +40,7 @@ services:
   cygnus-ld:
     hostname: cygnus-ld
     container_name: cygnus-ld
-    image: ging/fiware-cygnus
+    image: ging/cygnus-ngsi-ld
```

I understand this should be:

```suggestion
    image: fiware/cygnus-ngsi-ld
```